CONTRIBUTING.md

```diff
@@ -202,3 +202,18 @@ New dependency: dependency_name
 license: SPDX license tag
 origin: https://dependency_origin.com
 ```
+## Code coverage
+
+After adding new functionality add tests and check coverage before and after change.
+To do that, enable coverage instrumentation by enabling USE_GCOV flag in CMake.
+Example flow may look like:
```

Comment:
"To do this, enable coverage instrumentation by turning on the USE_GCOV flag in CMake.
An example flow might look like the following:"

Reply:
Done